packages/frontend/src/common/components/auth/Protected.tsx

```diff
@@ -0,0 +1,36 @@
+'use client'
+import React, { ReactNode, useEffect, useState } from 'react'
+import { getUser } from '@/api'
+import { Unauthorized } from '@/common/components/auth/Unauthorized'
+
+interface Props {
+  children: ReactNode
+}
+
+interface User {
+  id?: string
+  message?: string
```

Comment:
How do we use `message`?

Reply:
It was some experiments. I forgot to remove it.